pkg/e2e/operators/cloudingress/rhapi_lb.go

```diff
@@ -93,7 +122,136 @@ func testLBDeletion(h *helper.H) {
 					return false, nil
 				})
 				Expect(err).NotTo(HaveOccurred())
-			}
-		})
+
+			})
+		}
+
+		if viper.GetString(config.CloudProvider.CloudProviderID) == "gcp" {
+			util.GinkgoIt("Manually deleted LB should be recreated in GCP", func() {
+
+				region := viper.GetString("cloudProvider.region")
+				ctx := context.TODO()
+
+				ginkgo.By("Getting rh-api IP")
+				oldLBIP, err := getLBForService(h, "openshift-kube-apiserver", "rh-api", "ip")
+				Expect(err).NotTo(HaveOccurred())
+				fmt.Printf("old LB IP:  %s ", oldLBIP)
```

Comment:
There's a mixture of `fmt.Printf` and `log.Printf` throughout the code, might be best to standardize on `log.Printf` given other e2e code mostly uses that?

Reply:
updated